src/main/java/com/poiji/option/PoijiOptions.java

```diff
@@ -626,8 +639,18 @@ public PoijiOptionsBuilder poijiNumberFormat(final PoijiNumberFormat numberForma
 
         /**
          * Set true if all headers named in {@link ExcelCellName} are mandatory, otherwise false
+         * This feature needs headerstart and skip as mandatory parameters
+         * @param columnsToKeep the headers that should be included
+         */
+        public PoijiOptionsBuilder columnsToKeep(List columnsToKeep) {
+            this.columnsToKeep = columnsToKeep;
+            return this;
+        }
+
+        /**
+         * Provide columns to keep in case of repeated headers
```

Comment:
when an excel file contains repeated columns? in which cases ?

Reply:
Sorry, I think this line creeped in from somewhere by itself